design-documents/services-decomposition-guidelines.md

```diff
@@ -0,0 +1,45 @@
+# Services decomposition guidelines
+
+## About
+
+Document contains general recommendations and principles that need be considered and followed during services decomposition.
+
+## Guidelines
+
+1. Services should have it's own config_data table that would contain settings specific to the service. Services should expose API to modify settings in config_data. Admin panel should support editing settings in the local database (settings for BFF) and for individual services via remote calls.
```

Comment:
For me is better to move configuration in separate service like etcd, zookeeper.

Reply:
What would be the benefit of using something like etcd or ZooKeeper for storing configuration that user specifies in admin?

Reply:
Removed this item for now.